21/Dockerfile

```diff
@@ -51,11 +51,11 @@ RUN set -xe \
 	&& install -v ./rebar /usr/local/bin/ \
 	&& rm -rf /usr/src/rebar-src
 
-ENV REBAR3_VERSION="3.6.0"
+ENV REBAR3_VERSION="3.6.1"
 
 RUN set -xe \
 	&& REBAR3_DOWNLOAD_URL="https://github.com/erlang/rebar3/archive/${REBAR3_VERSION}.tar.gz" \
-	&& REBAR3_DOWNLOAD_SHA256="f36d7ee852ccb0dfcfec9b495ff2a29015549410acb97bf0c2377b90d7064ed1" \
+	&& REBAR3_DOWNLOAD_SHA256="40b3c85440f3235c7b149578d0211bdf57d1c66390f888bb771704f8abc710330c2377b90d7064ed1" \
```

Comment:
the sha256 doesn't look like very legit, how can the length change?   a valid sha256 should be length of 256bits, or in 64 hexdigits

Reply:
my calculation of `rebar3-3.6.1.tar.gz` is  `40b3c85440f3235c7b149578d0211bdf57d1c66390f888bb771704f8abc71033`